scripts/mass_user_emails.py

```diff
@@ -129,17 +168,23 @@ def run(*args):
 
     for user in eligible_users.iterator(chunk_size=500):
         for attempts in range(MAX_SEND_ATTEMPTS):
-            response = send_email(
-                ses, user.email, configuration=configuration_set
-            )
-            status = response['ResponseMetadata']['HTTPStatusCode']
+            try:
+                response = send_email(
+                    ses, user.email, configuration=configuration_set
```

Comment:
`user.email_cleaned` is available here thanks to your annotation, and it might be nice to use it instead of an arbitrarily-chosen `user+arbitrary@domain`. However, to do that, the regex you send to PostgreSQL needs to be adjusted (see next comment)